### PR TITLE
adding lit integrated mode for baremetal setup

### PR DESCRIPTION
**Pull request by TrezorHannes**
Adding support for users running LND integrated in LIT.

- Still need to testrun in my environment, but would appreciate an early review
- Need validating how integrated LND in LIT is built in Raspiblitz and Umbrel. For now, I left those integrations unchanged

Thanks

### CODE REVIEW

File: scripts/setupv2.sh [hunk 1]
```diff
@@ -96,24 +98,54 @@ done
...
-    echo "> Setting up Tunneling for CLN on port 9735 "
-    echo
-    lnImplementation="cln"
-    break
-    ;;
+  echo "Which lightning implementation do you want to tunnel?"
+  echo "    1) LND"
+  echo "    2) CLN (Core Lightning)"
+  if [ $litpossible -eq 1 ]; then
+    echo "    3) LIT"
```
Comment (blckbx) [inline scripts/setupv2.sh:105]:
```suggestion
    echo "    3) LIT (integrated mode)"
```

shall we make it clear that this requires LiT running in "integrated mode"?

**Review by blckbx — APPROVED** [review]
LGTM, only one clarification to consider

**Review by blckbx — DISMISSED** [review]

File: scripts/setupv2.sh [hunk 2]
```diff
@@ -8,8 +8,8 @@
 #Update if you make a significant change
 ##########UPDATE IF YOU MAKE A NEW RELEASE#############
 major=0
-minor=2
-patch=29
+minor=1
```
Comment (TrezorHannes) [inline scripts/setupv2.sh:11]: @blckbx I think we agreed that this is a patch only, do we have consensus on this?

File: scripts/setupv2.sh [hunk 3]
```diff
@@ -1375,8 +1374,8 @@ tor.skip-proxy-for-clearnet-targets=true
 fi
 
 if [ "$lnImplementation" == "cln" ]; then
-  if [ $isUmbrel -eq 1 ]; then
```
Comment (TrezorHannes) [inline scripts/setupv2.sh:1280]: My merge discarded this line but I think it needs to be in, would you mind reviewing and let me know @blckbx

File: scripts/setupv2.sh [hunk 4]
```diff
@@ -96,27 +98,54 @@ done
...
+      fi
+      ;;
+    4)
+      if [ $litpossible -eq 1 ]; then
+        echo "Exiting..."
+        exit 0
+      else
+        echo "Invalid option. Please enter a valid number."
+      fi
+      ;;
```
Comment (blckbx) [inline scripts/setupv2.sh:145]: `$litpossible` check seems redundant to me. it's done when options are being displayed. I think we can `exit 0` here without further checks.

File: scripts/setupv2.sh [hunk 5]
```diff
@@ -1276,9 +1352,30 @@ tor.skip-proxy-for-clearnet-targets=true
...
+[Tor]
+tor.streamisolation=false
+tor.skip-proxy-for-clearnet-targets=true
+#########################################"
+  echo
+
+fi
+
 if [ "$lnImplementation" == "cln" ]; then
-  if [ $isUmbrel -eq 1 ]; then
```
Comment (blckbx) [inline scripts/setupv2.sh:1280]: IF clause is required to distinguish between outputs

**Review by blckbx — CHANGES_REQUESTED** [review]

File: scripts/setupv2.sh (hunk 5 above)
Comment (TrezorHannes) [reply scripts/setupv2.sh:1280]: Ack I'll get this back in

File: scripts/setupv2.sh (hunk 4 above)
Comment (TrezorHannes) [reply scripts/setupv2.sh:145]: Agreed

**Review by blckbx — APPROVED** [review]

**Pull request merged**